### PR TITLE
PostmarketOS Galaxy Joke Post

### PR DESCRIPTION
**Pull request by mtekman**
It might be a bit too long, and also might need some more emojis

### CODE REVIEW

**Comment by hexylena** [comment]
absolute riot, i love it.

File: news/_posts/2024-01-31-postmarketos.md
[the remark this answers is not in the document]
Comment (mtekman) [reply news/_posts/2024-01-31-postmarketos.md:1]: I was thinking this too actually haha -- I don't mind, I leave this up to you :-D

File: news/_posts/2024-01-31-postmarketos.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+---
+title: "🪐📲 Hosting Galaxy at the Edge: Directly in Your Pocket!"
+tags: [fun, system administrators, developers, humour]
+contributions:
+  authorship: [mtekman]
```
Comment (mtekman) [inline news/_posts/2024-01-31-postmarketos.md:5]: @hexylena please add yourself here!

**Comment by mtekman** [comment]
can we post this before the weekend? I want to surprise some students of mine

**Comment by bgruening** [comment]
Thanks @mtekman!

**Review by bgruening — APPROVED** [review]

**Pull request merged**